prov/util/src/util_coll.c

```diff
@@ -1178,31 +1178,37 @@ int ofi_av_set(struct fid_av *av, struct fi_av_set_attr *attr,
 {
 	struct util_av *util_av = container_of(av, struct util_av, av_fid);
 	struct util_av_set *av_set;
-	int ret, iter;
+	uint64_t offset;
```

Comment:
nit: I would personally go with i vs offset, since it's a simple loop control index.